dashboard/client/src/components/StatusChip.tsx

```diff
@@ -23,71 +21,69 @@ import {
 } from "../type/serve";
 import { TypeTaskStatus } from "../type/task";
 
+const orange = "#DB6D00";
+
 const colorMap = {
   node: {
     ALIVE: green,
     DEAD: red,
   },
   worker: {
     ALIVE: green,
+    DEAD: red,
   },
   actor: {
     [ActorEnum.ALIVE]: green,
     [ActorEnum.DEAD]: red,
-    [ActorEnum.PENDING]: blue,
-    [ActorEnum.RECONSTRUCTING]: lightBlue,
+    [ActorEnum.DEPENDENCIES_UNREADY]: orange,
+    [ActorEnum.PENDING_CREATION]: orange,
+    [ActorEnum.RESTARTING]: orange,
   },
   task: {
     [TypeTaskStatus.FAILED]: red,
     [TypeTaskStatus.FINISHED]: green,
     [TypeTaskStatus.RUNNING]: blue,
-    [TypeTaskStatus.RUNNING_IN_RAY_GET]: blue,
-    [TypeTaskStatus.RUNNING_IN_RAY_WAIT]: blue,
-    [TypeTaskStatus.SUBMITTED_TO_WORKER]: "#cfcf08",
-    [TypeTaskStatus.PENDING_ARGS_FETCH]: blue,
-    [TypeTaskStatus.PENDING_OBJ_STORE_MEM_AVAIL]: blue,
-    [TypeTaskStatus.PENDING_NODE_ASSIGNMENT]: "#cfcf08",
-    [TypeTaskStatus.PENDING_ARGS_AVAIL]: "#f79e02",
+    [TypeTaskStatus.SUBMITTED_TO_WORKER]: orange,
+    [TypeTaskStatus.PENDING_NODE_ASSIGNMENT]: orange,
+    [TypeTaskStatus.PENDING_ARGS_AVAIL]: orange,
+    [TypeTaskStatus.NIL]: grey,
```

Comment:
Don't we need a mapping from the task types to one of these tasks? Like the mapping we have in the ProgressBar?